buildmaster/CMS_ttBar_lj_8TeV_dif/metadata.yaml

```diff
@@ -0,0 +1,120 @@
+# implemented by Tanishq Sharma
+
+setname: "CMS_ttBar_lj_8TeV_dif"
```

Comment:
if you look here https://github.com/NNPDF/pinecards and you want to be consistent with that, I think the name should be `CMS_TTBAR_8TEV_LJ` (maybe with an additional diff at the end?) cc @cschwan

Reply:
so <exp. name>_<process name>_<energy>_<channel>_<misc. info> is the format that should be used, correct? @felixhekhorn

Reply:
so `<exp. name>_<process name>_<energy>_<channel>_<misc. info>` is the format that should be used, correct? @felixhekhorn

Reply:
@t7phy that's exactly how I proposed it here: <https://github.com/NNPDF/pinecards/issues/82>.

Reply:
I think we also want all capital letters, don't we?

Reply:
> I think we also want all capital letters, don't we?

Yes, let's use capital letters everywhere. Beyond stylistic considerations, APFS which mac uses is case-sensitive depending on the day of the week [source](https://developer.apple.com/library/archive/documentation/FileManagement/Conceptual/APFS_Guide/FAQ/FAQ.html) . Thiss already created problems for me when updating the cfactors, which were working in my mac but not in my pc and had me very confused until I realised...

So please, let's use uppercase everywhere!